src/main/java/schmidt/core/Schmidt.java

```diff
@@ -0,0 +1,307 @@
+                    print(e.getMessage());
+                }
+                continue;
+            case "event":
+                try {
+                    addEventHelper(tasks, trimmedInput);
+                } catch (SchmidtException e) {
+                    print(e.getMessage());
+                }
+                continue;
+            default:
+                printValidCommands();
+            }
+        }
+    }
+
+    /**
+     * This is a function to add lines to print statements
+     */
+    public static void print(String message) {
+        System.out.println("------------------------------------------------------------");
+        System.out.println(message);
+        System.out.println("------------------------------------------------------------");
+    }
+
+    /**
+     * This is a function to print the valid commands
+     */
+    public static void printValidCommands() {
+        print("I'm sorry, but the valid commands are:\n" +
+                "\tbye\n" +
+                "\tlist\n" +
+                "\ttodo <description>\n" +
+                "\tdeadline <description> /by <date>\n" +
+                "\tevent <description> /from <date> /to <date>\n" +
+                "\tmark <task number>\n" +
+                "\tunmark <task number>");
+    }
+
+    /**
+     * This is a function to print the bye message
+     */
+    public static void printByeMessage() {
+        print("Bye. Hope to see you again soon!");
+    }
+
+    /**
+     * This is a helper method to print the list of tasks
+     *
+     * @param tasks The list of tasks
+     */
+    public static void printListHelper(ArrayList<Task> tasks) {
+        if (tasks.isEmpty()) {
+            print("You have no tasks in the list");
+            return;
+        }
+
+        System.out.println("------------------------------------------------------------\n" +
+                "Here are the tasks in your list:");
+
+        for (int i = 0; i < tasks.size(); i++) {
+            System.out.println("\t" + (i + 1) + ". " + tasks.get(i).toString());
+        }
+
+        System.out.println("------------------------------------------------------------");
+    }
+
+    /**
+     * This is a helper method to print the message after adding a task
+     *
+     * @param tasks The list of tasks
+     */
+    public static void printAddedTaskMessage(ArrayList<Task> tasks) {
+        print("Got it. I've added this task:\n" +
+                "\t" + tasks.get(tasks.size() - 1).toString() + "\n" +
+                "Now you have " + tasks.size() + " tasks in the list.");
+    }
+
+    /**
+     * This is a helper method to add a Todo task
+     *
+     * @param tasks The list of tasks
+     * @param input The user input
+     */
+    public static void addTodoHelper(ArrayList<Task> tasks, String input) {
+        // split by the first whitespace to get the description
+        String[] inputTokens = input.split("\\s+", 2);
+        if (inputTokens.length < 2) {
```

Comment:
Try to avoid magic numbers like this, maybe assigning this to a variable that provides contextual information would be better, i.e. `inputTokens.length < MIN_TOKENS_ADD_TODO`. This provides more contextual information about to someone reading the code